### DISCUSSION TITLE
magnetify french micro SAAS

### DISCUSSION THREAD

**Discussion by cyril-belin** [zainulabedeen123/best-saas-kit#7] [Show and tell]
hello 

here's one  saas made with best SAAS kit pro 
it's a lead magnet generator all working with some bug I have to fix but the main function is working I'm missing stripe configuration for subscription and landing page 

![Capture d’écran 2024-12-16 à 09 24 18](https://github.com/user-attachments/assets/b15368ab-4d39-4ca5-9b85-d17d3f64f77f)
![Capture d’écran 2024-12-16 à 09 24 46](https://github.com/user-attachments/assets/a3b40714-1d58-4eb9-a85e-02751ef3d0e5)
![Capture d’écran 2024-12-16 à 09 25 03](https://github.com/user-attachments/assets/4432bd8e-edd8-4bb0-a93d-dbfb95e67fab)
![Capture d’écran 2024-12-16 à 09 29 17](https://github.com/user-attachments/assets/0f87ccdf-4ca8-42a6-81c5-28b98f0a0ee3)
![Capture d’écran 2024-12-16 à 09 29 53](https://github.com/user-attachments/assets/e603fe42-21a4-4cff-961f-9cfea642dcd9)
![Capture d’écran 2024-12-16 à 09 30 19](https://github.com/user-attachments/assets/a4169e7a-adc3-4e31-ab59-0038045e7994)

**Comment by zainulabedeen123** [comment]
Wow, this looks promising. Idea is really good as well.

**Reply by cyril-belin** [reply]
thank you very much

[a comment by an automated account is not shown]
